RPIPs/RPIP-43.md

```diff
@@ -88,7 +88,7 @@ into shares is defined in [RPIP-46](RPIP-46.md).
   The megapool MUST assume that any other funds present are rewards.
 - A Node Operator SHALL be able to distribute rewards from all validators in their
   megapool at once.
-- Rewards distribution MAY be temporarily blocked while validators are exiting or
+- Rewards distribution MAY! be temporarily blocked while validators are exiting or
```

Comment:
This should maybe be clarified. Attempt:

"Rewards distribution MAY (fail gracefully / revert) while validators are exiting or pending removal."